update.sh

```diff
@@ -25,11 +25,47 @@ declare -A extras=(
 	[fpm-alpine]=''
 )
 
+apcu_version="$(
+	git ls-remote --tags https://github.com/krakjoe/apcu.git \
+		| cut -d/ -f3 \
+		| grep -vE -- '-rc|-b' \
+		| sed -E 's/^v//' \
+		| sort -V \
+		| tail -1
+)"
+
+memcached_version="$(
+	git ls-remote --tags https://github.com/php-memcached-dev/php-memcached.git \
+		| cut -d/ -f3 \
+		| grep -vE -- '-rc|-b' \
+		| sed -E 's/^[rv]//' \
+		| sort -V \
+		| tail -1
+)"
+
+redis_version="$(
+	git ls-remote --tags https://github.com/phpredis/phpredis.git \
+		| cut -d/ -f3 \
+		| grep -viE '[a-z]' \
+		| tr -d '^{}' \
+		| sort -V \
+		| tail -1
+)"
+
+imagick_version="$(
+	git ls-remote --tags https://github.com/mkoppanen/imagick.git \
+		| cut -d/ -f3 \
+		| grep -viE '[a-z]' \
+		| tr -d '^{}' \
+		| sort -V \
+		| tail -1
+)"
+
 declare -A pecl_versions=(
```

Comment:
is there any reason to keep this array? we can use the vars directly

Reply:
Maybe we have to use multiple versions for different versions of PHP/NextCloud, like we did for PHP 5.

Reply:
maybe but how would you do this with the current data structure?
for php 5 there where two templates, see: https://github.com/nextcloud/docker/tree/10

Reply:
The current structure does indeed not use `php_version`, but you can easily expand it if necessary.